### PR TITLE
Convenient link to blocked workers

### PR DESCRIPTION
**Pull request by peregrineshahin**
Currently, there is no convenient way to get to the particular worker page to block/unblock after misbehaving workers are spotted in test_view Tasks. You would need to write the URL manually or search for that worker on the home page.

Make the approver-only anchor redirect to the worker management rather than the user page as the user page is irrelevant after we can block only the worker.

### CODE REVIEW

**Comment by vdbergh** [comment]
There should also be a convenient link to a user profile page (to get their email address). I think this PR is good, but I don't know any other way to get to the profile page of a user, without hacking the url.

**Comment by vdbergh** [comment]
I see. One can also search in the Event log for an event in which the user is source or target. Still it is a bit indirect.

Perhaps the "Contributors" or "Top month" pages could contain links to the user profile pages (for maintainers).

**Comment by peregrineshahin** [comment]
Okay, that's a great suggestion.

**Comment by peregrineshahin** [comment]
I will make a client js search inside this page to search for any text in a row also. so let's draft this until I come back to this

**Comment by peregrineshahin** [comment]
> I will make a client js search inside this page to search for any text in a row also. so let's draft this until I come back to this

done here https://github.com/official-stockfish/fishtest/pull/1867

**Comment by vdbergh** [comment]
Could you perhaps add a comment to `filterTable` about how to use it? I feel that if it is in `application.js` it should be reusable.

**Comment by ppigazzini** [comment]
DEV updated with #1866 and #1867

**Review by ppigazzini — APPROVED** [review]
Looks good on DEV

**Comment by ppigazzini** [comment]
PROD updated, thank you @peregrineshahin :)

**Comment by peregrineshahin** [comment]
> Could you perhaps add a comment to `filterTable` about how to use it? I feel that if it is in `application.js` it should be reusable.

Sorry I messaged the message before the merge, It's pretty straight forward but I will add comments in next PR's probably in a cleanup

**Pull request merged**
